src/components/HeaderText.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from "prop-types";
+
+const HeaderText = ({ text, wavyText, size, textColor }) => {
+  let sizeClass;
+
+  switch (size) {
+    case "h1":
+      sizeClass = "max-[620px]:text-4xl text-6xl";
+      break;
+    case "h2":
+      sizeClass = "text-5xl";
+      break;
+    case "h3":
+      sizeClass = "text-4xl";
+      break;
+    default:
+      sizeClass = "text-3xl";
+  }
+
+  const headerClass = `${sizeClass} ${
+    textColor === "white" ? "text-white" : "text-black"
+  }  font-bold relative flex-col`;
```

Comment:
There are two spaces after `}` it should be single space.

Reply:
Also, maybe take help from chatGPT, this code is really hard to read because of many nested conditions.

Reply:
Made those more easier to read now. Thanks